### PR TITLE
Milestone 1.0.6

### PR DESCRIPTION
**Pull request by tonifisler**
- #300 Fix Wide Image paragraph
- #299 set all content type to "not published" by default
- #176 remove article skills from template (waiting for images to re-add)
- #303 remove all image size restrictions
- fix styles for definition lists
- change base layout to 8 columns instead of 6
- update all modules
- try to fix #260 english features export issue

### CODE REVIEW

**Comment by tonifisler** [comment]
The name of my branch is not up-to-date. Don't pay attention. :grin:

**Comment by gido** [comment]
:+1: Looks good

File: drupal/sites/all/themes/antistatique/templates/node/node.tpl.php [hunk 1]
```diff
@@ -159,6 +159,7 @@
             <?php print render($content['field_skills']); ?>
           </div>
         <?php endif ?>
+        <?php /* ?>
```
Comment (gido) [inline drupal/sites/all/themes/antistatique/templates/node/node.tpl.php]: Add a comment with the github issue #306 (I don't like commented code)

**Comment by Yago** [comment]
check #309 in the same time before release

**Comment by tonifisler** [comment]
Merging! :shipit:

**Pull request merged**